core/controller/src/main/scala/whisk/core/controller/Meta.scala

```diff
@@ -62,11 +62,7 @@ private case class Context(
 }
 
 protected[core] object WhiskMetaApi extends Directives {
-    /**
-     * Defines the properties that must be present in a configuration
-     * in order to implement the Meta API.
-     */
-    def requiredProperties = Map(WhiskConfig.systemKey -> null)
+    def requiredProperties = Map()
```

Comment:
debatable - i removed it then changed my mind.
i cant articulate my reasons strongly, so i'll remove it again.